### PR TITLE
Linked list

### PR DESCRIPTION
**Pull request by littl3birdy**
[x ] Top-level README “Table of Contents” is updated
 Feature tasks for this challenge are completed
 Unit tests written and passing
 “Happy Path” - Expected outcome
 Expected failure
 Edge Case (if applicable/obvious)
[ x] README for this challenge is complete
[x ] Summary, Description, Approach & Efficiency, Solution
[ x] Link to code

### CODE REVIEW

File: code401challenges/src/main/java/code401challenges/LinkedList.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        Node next;
+        Node(int d){
+            data = d;
+            next = null;
+        }
+
+    }
+    public void insertHead(int value){
+        Node newHead = new Node(value);
+        this.head =newHead;
```
Comment (SharinaS) [inline code401challenges/src/main/java/code401challenges/LinkedList.java]: Here is actually where you can do a bit of if / else logic: if this list's head is null, set the head to be the new node; otherwise, set the head to be the new node, and set the new node's next to be the old head.

File: code401challenges/src/main/java/code401challenges/LinkedList.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        }
+
+    public void insert(int value){
+        Node newNode = new Node(value);
+        if(this.head == null) {
+            this.head = newNode;
+        }
+        else {
+            head.next= newNode;
```
Comment (SharinaS) [inline code401challenges/src/main/java/code401challenges/LinkedList.java]: To insert a node at the head, this is where you would write newNode.next = head; (so, switch what you have), and then, you need to set head to be the new node... so, head.next = newNode;

File: code401challenges/src/main/java/code401challenges/LinkedList.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        while(head != null) {
+            if(head.data == value){
+                return true;
+            }
+            head = head.next;
+        }
+
+        return false;
+    }
+
```
Comment (SharinaS) [inline code401challenges/src/main/java/code401challenges/LinkedList.java]: You may need an @Override, here, above your public String toString()
